packages/consumption/src/consumption/ConsumptionCoreErrors.ts

```diff
@@ -306,6 +306,10 @@ class Requests {
         return new CoreError("error.consumption.requests.missingRelationship", message);
     }
 
+    public inheritedFromItem(message: string) {
+        return new ApplicationError("error.consumption.validation.inheritedFromItem", message);
```

Comment:
`.requests` missing in the error code if you place the error here in the file under the class `Requests`, for example, `"error.consumption.requests.validation.inheritedFromItem"` instead of `"error.consumption.validation.inheritedFromItem"`.

Reply:
Yes that's true I didn't want to change the docu... But I will do it then.